bndtools.builder/src/org/bndtools/builder/BndtoolsBuilder.java

```diff
@@ -117,6 +119,7 @@
             if (model == null) {
                 try {
                     model = Central.getProject(myProject.getLocation().toFile());
+                    model.addBasicPlugin(new JobProgress());
```

Comment:
Yes, i just checked and the model that is returned already has a JobProgress plugin (probably inherited from workspace)